sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/FileDataSourceV2.scala

```diff
@@ -43,11 +43,10 @@ trait FileDataSourceV2 extends TableProvider with DataSourceRegister {
 
   protected def getPaths(map: CaseInsensitiveStringMap): Seq[String] = {
     val objectMapper = new ObjectMapper()
-    Option(map.get("paths")).map { pathStr =>
+    val paths = Option(map.get("paths")).map { pathStr =>
```

Comment:
NIT: you can convert the option to a Seq, so you don't need to the getOrElse call.

Reply:
The result of `.map { pathStr => objectMapper.readValue(pathStr, classOf[Array[String]]).toSeq` is Option[Seq[String]].  I think the getOrElse si fine here.